BUILD.md

```diff
@@ -203,15 +203,12 @@ Dockerfile instructions are available [here](./dockerfiles#tensorrt)
 
 ---
 
-### MKLDNN and MKLML
-See more information on MKL-DNN and MKL-ML [here](./docs/execution_providers/MKL-DNN-ExecutionProvider.md).
+### DNNL and MKLML
+See more information on DNNL and MKL-ML [here](./docs/execution_providers/MKLDNN-ExecutionProvider.md).
```

Comment:
Typo in the name of md file.
But maybe it would be good to rename it to DNNL-ExecutionProvider.md

Reply:
Fixed.